README.md

```diff
@@ -18,7 +18,16 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+To use with Devise, add the following to your Devise initializer (typically `devise.rb`):
+
+```ruby
+  config.omniauth :gds,
+    ENV["APPLICATION_ID"], # Obtained when generating a new application
```

Comment:
It may be an idea to use `ENV.fetch` in these examples, so they'd break loudly rather than return `nil` if not set?